src/javascript/class-scanner.ts

```diff
@@ -586,6 +586,80 @@ class CustomElementsDefineCallFinder implements Visitor {
   }
 }
 
+export function extractPropertyFromMethod(
+    method: babel.ClassMethod, document: JavaScriptDocument): ScannedProperty | null {
+  if (method.static || method.key.type !== 'Identifier') {
+    return null;
+  }
+
+  if (method.key.name === undefined) {
+    return null;
+  }
+
+  const annotation = getJSDocAnnotationForNode(method);
+  if (!annotation) {
+    return null;
```

Comment:
Do we want to emit nothing if there's no annotation? It's still definitely a property.

Reply:
this is what the previous behaviour was so im not sure. i think it was to avoid reading `/** */` as a property or some such thing... happy to change it if we no longer want that.